assets/src/components/GoogleAnalyticsTracking.js

```diff
@@ -26,4 +25,4 @@ function GoogleAnalyticsTracking (props) {
   return null
 }
 
-export default withRouter(GoogleAnalyticsTracking)
+export default GoogleAnalyticsTracking
```

Comment:
Not sure why `withRouter` was used here to begin with, since I'm not seeing any `react-router-dom` params (like `match` or `location`) be used in `props`...

Reply:
It's probably worth testing Google Analytics in test if we can, though.

Reply:
I agree on this. https://analytics.google.com/

Reply:
Mostly google analytics is real time update to dashboard. But Last time I used chrome extension if GA data is collected where we need to https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en

Reply:
This should work, I agree with you. But  see the GA action is performed using the chrome extension, But I struggled seeing data from GA official Website. But I don't feel this change has anything to do with that.